README.md

```diff
@@ -161,9 +120,9 @@ Updates `betterdiscordctl` to the latest version available on GitHub.
 
   Reinstalls BetterDiscord to Discord Canary.
 
-* `betterdiscordctl update --flatpak`
+* `betterdiscordctl status --flatpak`
```

Comment:
We already demonstrate `status`. I'd rather we just drop this example than bring up `Flatpak` in a contrived way.